.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+- React-Native tools version:
```

Comment:
Then, make the title less ambiguous. "React-native extension version"? Also, wouldn't react-native version be a necessary data point?